policyengine_us/parameters/gov/ssa/ssi/income/exclusions/blind_or_disabled_working_student/age_limit.yaml

```diff
@@ -0,0 +1,39 @@
+description: IRS limits the blind or disabled SSI amount to student filers at or below this age limit.
+
+values:
+  1974-01-01: 22
+metadata:
+  label: SSI blind or disabled working student earned income exclusion age limit
+  unit: year
+  period: year
+  reference:
+    - title: 20 CFR § 416.1112 - Earned income we do not count.
+      href: https://www.law.cornell.edu/cfr/text/20/416.1112#c_3
+    - title: 2011 Annual Report of the SSI Program - B. INCOME AND RESOURCE EXCLUSIONS
```

Comment:
appreciate these but for the future, we generally don't need non-USC/CFR sources for parameters not indexed to inflation (like these)